android/src/main/java/com/reactnativemembrane/MembraneWebRTCModule.kt

```diff
@@ -193,19 +193,19 @@ class MembraneWebRTCModule : Module() {
 
         AsyncFunction("updateVideoTrackMetadata") Coroutine { metadata: Map<String, Any> ->
             withContext(Dispatchers.Main) {
-                membraneWebRTC.updateVideoTrackMetadata(metadata)
+                membraneWebRTC.updateLocalVideoTrackMetadata(metadata)
             }
         }
 
         AsyncFunction("updateAudioTrackMetadata") Coroutine { metadata: Map<String, Any> ->
             withContext(Dispatchers.Main) {
-                membraneWebRTC.updateAudioTrackMetadata(metadata)
+                membraneWebRTC.updateLocalAudioTrackMetadata(metadata)
             }
         }
 
         AsyncFunction("updateScreencastTrackMetadata") Coroutine { metadata: Map<String, Any> ->
```

Comment:
Are you sure that discrepancy between AsyncFunction name and mambraneWebRTC.method is valid?

Reply:
Are you asking if the functions can have different names or if they should have different names? They can indeed have different names, and the function will still be correctly called. However, I agree that they should ideally have the same name. I opted to change the name for clarity in the native code and didn't want to introduce a breaking change to React Native

Reply:
If you believe this is a bad idea, please let me know 😊